### DISCUSSION TITLE
Controller No Longer Showing Info

### DISCUSSION THREAD

**Discussion by vmcosco** [zwave-js/zwave-js-ui#2310] [Q&A] [answered]
Since the most recent update, the ZWave controller is no longer showing "Status" or most of the other fields.  Is this expected or did something maybe happen with my stick?  All nodes are functioning properly.   

I can confirm that the Status for my controller showed as "alive" until I updated the last time.

**Comment by robertsLando** [comment, marked as answer]
@vmcosco Some columns (like status) have been hidden for controller as there is no reason to show them :)

**Reply by vmcosco** [reply]
That makes perfect sense.  Just wanted to make sure that something wasn't going on.   I only noticed it because I have a sensor setup in HA that counts "alive" and "sleeping" nodes and compares it to total nodes so I can quickly see if any nodes are "unavailable" or "dead".  The node status for my controller went to unavailable so it triggered an alert on the sensor.

**Reply by kpine** [reply]
FYI, the controller node status entity went to unavailable because it was removed in 2022.3, for the same reason. You can just delete the entity.
